packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.tsx

```diff
@@ -107,13 +100,6 @@ export const CertificateActionContent: TCertificateActionContent = ({
           </>
         )}
       </Button>
-      {selectedIds.length > 1 && (
```

Comment:
Have you checked if the bulk actions work with all the available actions of both Exchange and Blockchain inboxes ?

Reply:
I have tested this for Blockchain inbox (locally with D-REC) cannot test in Origin. Need your help here.

Reply:
Ok, will check it